bot.py

```diff
@@ -50,6 +52,21 @@
     else:
         await client.send_message(message.channel, 'The current song is : "{}"'.format(elevator.title))
 
+async def playSound(message, soundURL):
+    if not in_elevator:
+        voice_channel = client.get_channel(message.author.voice.voice_channel.id)
+        voice_client = await client.join_voice_channel(voice_channel)
+        genji = await voice_client.create_ytdl_player(soundURL)
+        genji.volume = .2
```

Comment:
One-shot sound effects should probably be a higher volume. It's the background music we want to be quiet so that people can talk over it.